standard-page.js

```diff
@@ -144,6 +162,24 @@ H5P.StandardPage = (function ($, EventDispatcher) {
     });
   };
 
+  /**
+   * Sets previous state values for input fields
+   * @param state
+   */
+  StandardPage.prototype.setPreviousState = function (state) {
+    var inputIndex = 0;
+
+    this.pageInstances.forEach(function (instance) {
+      if (instance.libraryInfo.machineName === 'H5P.TextInputField' && instance.$inputField !== undefined) {
```

Comment:
This feels wrong. A better solution would be to implement getCurrentstate in H5P.TextInputField